jest.config.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+module.exports = {
+  clearMocks: true,
+  cacheDirectory: ".cache/jest",
+  collectCoverage: true,
+  coverageDirectory: "./coverage",
+  coverageReporters: ["html", "json", "text-summary"],
+  testEnvironment: "node",
```

Comment:
```suggestion

```

Do we need to be explicit with our testEnvironment, or can we rely on the Jest 27 default to `node`?

Reply:
Sure, we can rely on default.